library/src/main/java/com/anjlab/android/iab/v3/SkuDetails.java

```diff
@@ -83,72 +77,65 @@ public SkuDetails(JSONObject source) throws JSONException
         priceLong = source.optLong(Constants.RESPONSE_PRICE_MICROS);
         priceValue = priceLong / 1000000d;
         priceText = source.optString(Constants.RESPONSE_PRICE);
-        subscriptionPeriod = parsePeriod(source.optString(Constants.RESPONSE_SUBSCRIPTION_PERIOD, "P"));
-        subscriptionFreeTrialPeriod = parsePeriod(source.optString(Constants.RESPONSE_FREE_TRIAL_PERIOD, "P"));
-        haveTrialPeriod = subscriptionFreeTrialPeriod.toStandardSeconds().getSeconds() != 0;
+        subscriptionPeriod = source.optString(Constants.RESPONSE_SUBSCRIPTION_PERIOD, "P");
```

Comment:
Also here a ref. of how a subscriptionPeriod looks like: https://github.com/anjlab/android-inapp-billing-v3/pull/281/files#diff-ebc6d11b564dc13ba02eb68f29e058ffR9